CMakeLists.txt

```diff
@@ -41,6 +41,11 @@ common_find_package(Qt5WebKitWidgets SYSTEM)
 common_find_package(Qt5Widgets REQUIRED)
 common_find_package(Qt5XmlPatterns REQUIRED)
 common_find_package(Qt5Xml REQUIRED)
+common_find_package(ZeroEQ)
+
+if(ZeroEQ_FOUND)
```

Comment:
I have no preference, but it's my understanding that TARGETs are the cmake 3 way, and Find/FOUND is not advised.